pkgs/development/python-modules/python-can-isotp/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  autopep8,
+  coverage,
+  ipdb,
+  mypy,
+  pytestCheckHook,
+  python-can,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "python-can-isotp";
```

```suggestion
  pname = "can-isotp";
```
> Attribute names in python-packages.nix as well as pnames should match the library's name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names). This means that characters should be converted to lowercase and . and _ should be replaced by a single - (foo-bar-baz instead of Foo__Bar.baz).

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines